doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -206,21 +92,38 @@ General Purpose Equivalents
    :header-rows: 1
 
    * - **MATLAB**
-     - **numpy**
+     - **NumPy**
```

Comment:
That's true, we shouldn't encourage `import *`. 

I think it might be better to add `np.<func>` to the table. It will also help to separate `scipy.linalg` from `numpy.linalg` since there are overlapping functions.

Reply:
Latest commit removed the `import *` on L82. 

replaced those lines with 

```python
    import numpy as np
    from scipy import io, integrate, linalg, signal
    from scipy.sparse.linalg import eigs
```
commit 2812833bc5f1bb214ae86045e4582ceeb4b464dd